Frameworks/Foundation/NSMutableSet.mm

```diff
@@ -139,11 +139,21 @@ - (void)removeObject:(id)object {
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
 - (void)filterUsingPredicate:(NSPredicate*)predicate {
-    UNIMPLEMENTED();
+    if (predicate == NULL) {
+        return;
+    }
+    @synchronized(self) {
+        NSArray* objects = [self allObjects];
```

Comment:
Nope it's not. i'll go ahead with the removal. I thought i can guard the copy->removal phase

---
In reply to: [74295924](https://github.com/Microsoft/WinObjC/pull/857#discussion_r74295924) [](ancestors = 74295924)